_episodes/08-putting-it-all-together.md

```diff
@@ -86,209 +89,218 @@ also change the name of the columns in the DataFrame like this:
 ~~~
 {: .language-python}
 
-## Make a line plot of your data
-
-Matplotlib is a Python library that can be used to visualize data. The
-toolbox `matplotlib.pyplot` is a collection of functions that make matplotlib
-work like MATLAB. In most cases, this is all that you will need to use, but
-there are many other useful tools in matplotlib that you should explore.
-
-We will cover a few basic commands for formatting plots in this lesson. A great
-resource for help styling your figures is the matplotlib gallery
-(http://matplotlib.org/gallery.html), which includes plots in many different
-styles and the source code that creates them. The simplest of plots is the 2
-dimensional line plot. These examples walk through the basic commands for making
-line plots using pyplots.
-
-> ## Challenge - Lots of plots
-> Make a variety of line plots from your data. If you are using the streamgage
-> data, these could include (1) a hydrograph of the entire month of September
-> 2013, (2) the discharge record for the week of the 2013 Front Range flood
-> (September 9 through 15), (3) discharge vs. time of day, for every day in the
-> record in one figure (Hint: use loops to combine strings and give every line a
-> different style and color), and (4) minimum, maximum, and mean daily discharge
-> values. Add axis labels, titles, and legends to your figures. Make at least one
-> figure with multiple plots using the function `subplot()`.
-{: .challenge}
+## Matplotlib package
 
-### Using pyplot:
+[Matplotlib](http://matplotlib.org/) is a Python package used widely throughout the scientific Python community to produce high-quality and publication-ready graphics. It supports a wide range of output formats including PNG (and other raster formats), PostScript/EPS, PDF and SVG. 
 
-First, import the pyplot toolbox:
+At the same time, matplotlib is the actual engine behind the plotting capabilities of both the Pandas and the plotnine package. When you called the `.plot` functionality of Pandas, as we did in the previous episodes, you actually used the matplotlib package:
 
 ~~~
-    import matplotlib.pyplot as plt
+import pandas as pd
+surveys = pd.read_csv("./data/surveys.csv")
```

Comment:
previous lessons just do data/surveys.csv, would you mind switching to that?

Reply:
consistency matter, I'll fix it